### PR TITLE
Feature/feature remote device

### PR DESCRIPTION
**Pull request by hhou2011**
@Pacane

### CODE REVIEW

File: src/main/java/org/dsa/iot/knx/EditablePoint.java [hunk 1]
```diff
@@ -27,17 +27,14 @@
...
-	static final String ATTR_SUB_GROUP_ADDRESS = "sub group address";
 	static final String ATTR_INDIVIDUAL_ADDRESS = "individual address";
 	static final String ATTR_RESTORE_TYPE = "restore type";
 	static final String RESTORE_EDITABLE_POINT = "editable point";
 
 	static final String ACTION_REMOVE = "remove";
 	static final String ACTION_EDIT = "edit";
 
-	static final String DEFAULT_GROUP_ADDRESS = "0";
+	static final String DEFAULT_ADDRESS = "0.0.0";
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/EditablePoint.java:37]: that looks erroneous. Shouldn't it be `0.0.0.0` ?

File: src/main/java/org/dsa/iot/knx/KnxLink.java [hunk 2]
```diff
@@ -42,7 +42,7 @@
 	static final int DEFAULT_POLLING_INTERVAL = 5;
 	static final int DEFAULT_POLLING_TIMEOUT = 5;
 	static final String DEFAULT_MULTICAST_ADDRESS = "";
-
+	static final String DEFAULT_DEVICE_ADDRESS = "0.0.0";
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/KnxLink.java:45]: same here?

File: src/main/java/org/dsa/iot/knx/datapoint/DPT2ByteFloat.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+		super(dptId);
+		this.format = format;
+		this.suffix = suffix;
+	}
+
+	@Override
+	public int getDataTypeId() {
+		return DataTypes.NUMERIC.ordinal();
+	}
+
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/datapoint/DPT2ByteFloat.java:19]: rm empty line

File: src/main/java/org/dsa/iot/knx/datapoint/DPT4ByteSigned.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+		super(dptId);
+		this.format = format;
+		this.suffix = suffix;
+	}
+
+	@Override
+	public int getDataTypeId() {
+		return DataTypes.NUMERIC.ordinal();
+	}
+
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/datapoint/DPT4ByteSigned.java:19]: rm empty line

File: src/main/java/org/dsa/iot/knx/datapoint/DPT8BitUnsigned.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+		super(dptId);
+		this.format = format;
+		this.suffix = suffix;
+	}
+
+	@Override
+	public int getDataTypeId() {
+		return DataTypes.NUMERIC.ordinal();
+	}
+
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/datapoint/DPT8BitUnsigned.java:19]: rm empty line

File: src/main/java/org/dsa/iot/knx/datapoint/DPTString.java [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+public class DPTString extends DPT {
+	public DPTString(String dptId) {
+		super(dptId);
+	}
+
+	@Override
+	public int getDataTypeId() {
+		return DataTypes.ALPHANUMERIC.ordinal();
+	}
+
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/datapoint/DPTString.java:14]: rm

File: src/main/java/org/dsa/iot/knx/datapoint/DatapointType.java [hunk 7]
```diff
@@ -36,8 +36,218 @@
...
+    
+    //   CONTROL_DIMMING("knx.dpt.control.dimming", DPTXlator3BitControlled.class, "3.007", null), //
+    //   CONTROL_BLINDS("knx.dpt.control.blinds", DPTXlator3BitControlled.class, "3.008", null), //
+
+    //
+    //
+    // 8 bit unsigned (5)
+    //
+    EIGHTBITUNSIGNED_SCALING("knx.dpt.8bitu.scaling", "DPST-5-1", new DPT8BitUnsigned("5.001", "#.#", "%")), //
+    EIGHTBITUNSIGNED_ANGLE("knx.dpt.8bitu.angle", "DPST-5-3", new DPT8BitUnsigned("5.003", "#.#", "\u00b0")), //
```
Comment (Pacane) [inline src/main/java/org/dsa/iot/knx/datapoint/DatapointType.java:51]: the comments in the end of those lines are useless

File: src/main/java/org/dsa/iot/knx/EditablePoint.java (hunk 1 above)
Comment (hhou2011) [inline src/main/java/org/dsa/iot/knx/EditablePoint.java:37]: That's not ip address.  That is knx group address.

File: src/main/java/org/dsa/iot/knx/KnxLink.java (hunk 2 above)
Comment (hhou2011) [inline src/main/java/org/dsa/iot/knx/KnxLink.java:45]: Either knx device or group address has three levels.

**Pull request merged**